web/src/routes/PoisPage.tsx

```diff
@@ -145,22 +169,36 @@ const PoisPage = ({ cityCode, languageCode, cityModel, pathname, languages }: Ci
       </LocationLayout>
     )
   }
-  const sortedPois = pois.sort((poi1: PoiModel, poi2: PoiModel) => poi1.title.localeCompare(poi2.title))
-  const renderPoiListItem = (poi: PoiModel) => <PoiListItem key={poi.path} poi={poi} />
+  const sortedPois = featureLocations?.sort((poi1: PoiFeature, poi2: PoiFeature) =>
+    poi1.properties.title.localeCompare(poi2.properties.title)
+  )
+  const renderPoiListItem = (poi: PoiFeature) => <PoiListItem key={poi.properties.path} properties={poi.properties} />
   const pageTitle = `${t('pageTitle')} - ${cityModel.name}`
-  const featureLocations = pois.map(poi => poi.featureLocation).filter((feature): feature is PoiFeature => !!feature)
 
   return (
     <LocationLayout isLoading={false} {...locationLayoutParams}>
       <Helmet pageTitle={pageTitle} languageChangePaths={languageChangePaths} cityModel={cityModel} />
-      <Caption title={t('pois')} />
       {cityModel.boundingBox && (
         <MapView
-          featureCollection={embedInCollection(featureLocations)}
+          featureCollection={featureLocations && embedInCollection(featureLocations)}
           bboxViewport={moveViewToBBox(cityModel.boundingBox, defaultViewportConfig)}
+          ref={sheetRef}
+          currentFeature={currentFeature}
+          setCurrentFeature={setCurrentFeature}
         />
       )}
-      <List noItemsMessage={t('noPois')} items={sortedPois} renderItem={renderPoiListItem} />
+      <BottomActionSheet title={currentFeature ? '' : t('sheetTitle')} ref={sheetRef}>
+        {currentFeature ? (
+          <div>{currentFeature.properties.title}</div>
+        ) : (
+          sortedPois && (
+            <ListWrapper>
+              <List noItemsMessage={t('noPois')} items={sortedPois} renderItem={renderPoiListItem} borderless />
+            </ListWrapper>
+          )
+        )}
```

Comment:
Make title optional in the bottom sheet and use the following:
```suggestion
      <BottomActionSheet title={currentFeature?.properties.title} ref={sheetRef}>
        {sortedPois && !currentFeature && (
          <ListWrapper>
            <List noItemsMessage={t('noPois')} items={sortedPois} renderItem={renderPoiListItem} borderless />
          </ListWrapper>
        )}
```
Or any specific reason why you don't use the title for feature titles?

I think `Entdecke deine Stadt` is nothing that should be shown (it is also not shown in native and is removed from the translations in #541).

Reply:
because i didn't implement the details view yet for the bottom sheet ;)
your change in line 190 makes sense. The rest not because i would like to keep the conditional rendering for the details view which comes up with another ticket